build.gradle

```diff
@@ -33,11 +33,11 @@ configurations {
 }
 
 dependencies {
-    shade 'fr.inria.gforge.spoon:spoon-core:5.7.0'
+    shade 'fr.inria.gforge.spoon:spoon-core:5.5.0'
```

Comment:
Is there a specific reason for why you downgraded the Spoon dependency?

Reply:
Has the spoon issue been fixed since 5.7 now that they've had quite a few releases since then? What even was the 🥄 issue?

Reply:
Nope, they simply changed the behavior. Unfortunately, I do not understand spoon well enough to update code in event impl gen.

Reply:
I'm not sure I understand.

> The current version of event-impl-gen that sponge api uses is the latest release instead of the snapshot

But it's clearly specified here at 5.7.0 ? Why would it use the latest instead of the snapshot? Why would we **want** to use the snapshot over the version listed? and 5.5.0 isn't the snapshot?

What behavior changed?